### PR TITLE
Renamed to snake case

### LINKED ISSUE #84: Usage of snake_case
Make sure all deepworlds variables follow the snake_case convention.

### PR DESCRIPTION
**Pull request by KelvinYang0320**
- [x] cartpole
  - [x] cartpole_continuous
    - Verifying
      `xvfb-run webots --stdout --stderr --batch  --mode=fast ./deepworlds/examples/cartpole/cartpole_continuous/worlds/cartpole_world_robot_supervisor.wbt`
      `xvfb-run webots --stdout --stderr --batch  --mode=fast ./deepworlds/examples/cartpole/cartpole_continuous/worlds/cartpole_world_emitter_receiver.wbt`
  - [x] cartpole_discrete
    - Verifying
      `xvfb-run webots --stdout --stderr --batch  --mode=fast ./deepworlds/examples/cartpole/cartpole_discrete/worlds/cartpole_world_robot_supervisor_stable_baselines.wbt`
      `xvfb-run webots --stdout --stderr --batch  --mode=fast ./deepworlds/examples/cartpole/cartpole_discrete/worlds/cartpole_world_robot_supervisor.wbt`
      `xvfb-run webots --stdout --stderr --batch  --mode=fast ./deepworlds/examples/cartpole/cartpole_discrete/worlds/cartpole_world_emitter_receiver.wbt`
- [x] find_and_avoid
    - Verifying
      `xvfb-run webots --stdout --stderr --batch  --mode=fast ./deepworlds/examples/find_and_avoid/worlds/small_world.wbt`
- [x] khr-3hv (it has already followed snake case)
- [x] panda
    - Verifying
      `xvfb-run webots --stdout --stderr --batch  --mode=fast ./deepworlds/examples/panda/panda_goal_reaching/worlds/panda_goal_reaching.wbt`
- [x] pit_escape
    - Verifying
      `xvfb-run webots --stdout --stderr --batch  --mode=fast ./deepworlds/examples/pit_escape/pit_escape_discrete/worlds/pit_escape.wbt`

### CODE REVIEW

File: examples/cartpole/cartpole_continuous/controllers/robot_supervisor_manager/utilities.py [hunk 1]
```diff
@@ -2,62 +2,62 @@
...
-    :param saveName: Filename of saved plot, defaults to None
-    :type saveName: str, optional
+    :param save_name: Filename of saved plot, defaults to None
+    :type save_name: str, optional
     """
     fig, ax = plt.subplots()
     ax.plot(data)
-    ax.set(xlabel=xLabel, ylabel=yLabel,
-           title=plotTitle)
+    ax.set(x_label=x_label, y_label=y_label,
```
Comment (tsampazk) [inline examples/cartpole/cartpole_continuous/controllers/robot_supervisor_manager/utilities.py]: Just caught this independently as i did the same mistake, argument names are xlabel, ylabel https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.set.html :laughing:

Comment (KelvinYang0320) [reply examples/cartpole/cartpole_continuous/controllers/robot_supervisor_manager/utilities.py]: @tsampazk Thanks! I didn't notice that.

**Review by tsampazk — DISMISSED** [review]
Looks good @KelvinYang0320, thank you!

**Comment by KelvinYang0320** [comment]
@tsampazk Could you approve it again? I just rebased and squashed all commits to one commit.

**Review by tsampazk — APPROVED** [review]

**Pull request merged**